.gitignore

```diff
@@ -26,5 +26,7 @@
 # Ignore .env from dot-env, used for local config
 .env
 
+# Ignore ds_store files from Mac
+.DS_Store
```

Comment:
There is a line of thinking that system-specific file ignores belong in one's global .gitignore file. Per-project .gitignore's then stay focused on keeping a specific project in order.

Reply:
That's a valid point @usufruct. I don't feel two strongly on this, so I am fine with this going in with the ignore, but @loudmouse if you want to learn how to apply this ignore globally, you can find some info [here](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore). I'll leave it up to you to decide whether you want to do it globally and remove it from this PR, or leave it here, since I think it has relatively low impact either way.